unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendsHUD/Scripts/Tabs/FriendsTabView.cs

```diff
@@ -137,4 +145,31 @@ private void ChatController_OnAddMessage(ChatMessage message)
 
         lastProcessedFriend = friend.userId;
     }
+
+    public void RequestCreateEntry(string userId, FriendEntryBase.Model model)
+    {
+        creationQueue[userId] = model;
```

Comment:
the eternal fight over `dic[newIndex]` vs `dic.Add(newIndex, value)` XD

Reply:
Oops, I completely forgot about this nonsense (:p). Gonna change it!

Reply:
Maybe we can discuss about this again. I have to admit the brackets expression approach looks good once you get used to it.